.travis.yml

```diff
@@ -7,6 +7,6 @@ cache:
   directories:
     - ~/.m2/repository
 install:
-  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -P travis
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true --quiet --batch-mode --show-version --activate-profiles coveralls
```

Comment:
Long options preferred?

Reply:
Always ;) So we don't have to refer to the docs to remember what the hell `-B` means etc.